cadc-gms/src/main/java/org/opencadc/auth/StandardIdentityManager.java

```diff
@@ -388,15 +381,7 @@ private void validateOidcAccessToken(Subject s) {
     
     private URL getUserEndpoint() {
```

Comment:
just remove this method entirely and replace it's use with the oidcClient call

Reply:
Done.